modules/gitops-creating-a-new-client-in-dex.adoc

```diff
@@ -4,27 +4,22 @@
 [id="gitops-creating-a-new-client-in-dex_{context}"]
 = Enabling the Dex OpenShift OAuth Connector
 
-Dex uses the users and groups defined within OpenShift by checking the `OAuth` server provided by the platform. The following example shows the properties of Dex along with example configurations:
+Dex is installed by default for all the Argo CD instances created by the Operator. You can configure {gitops-title} to use Dex as the SSO authentication provider by setting the `.spec.sso` parameter. 
 
+Dex uses the users and groups defined within OpenShift by checking the `OAuth` server provided by the platform. 
+
+.Procedure
+
+* To enable Dex, set the `.spec.sso.provider` parameter to `dex` in the YAML resource of the Operator:
++
 [source,yaml]
 ----
-apiVersion: argoproj.io/v1alpha1
-kind: ArgoCD
-metadata:
-  name: example-argocd
-  labels:
-    example: openshift-oauth
+...
```

Comment:
s/.../# ...   
See the [doc guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#formatting-1).